_CodingChallenges/102-2d-water-ripple.md

```diff
@@ -34,6 +33,12 @@ contributions:
       name: "Marc Benzahra"
     url: "https://media.giphy.com/media/5wG3oCLNct1h0wE5ML/giphy.gif"
     source: "https://github.com/1475963/be-water-my-friend"
+  - title: "Ripple in Processing with added functions and sound"
+    author:
+      name: "Jelle Teijema"
+      url: "https://jjteijema.nl"
```

Comment:
```suggestion
      url: "http://jjteijema.nl"
```
I'm sorry, but I do not want to bother you, but your website does not seem to support https

Reply:
Hahaha yeah I host for free and the ssl costs money. Thanks tho.

Reply:
@JTeijema Reminder :D When you push your update we will be able to merge.